.pre-commit-config.yaml

```diff
@@ -19,21 +19,20 @@ repos:
     rev: v1.7.0
     hooks:
       # See https://github.com/pre-commit/mirrors-mypy/blob/main/.pre-commit-hooks.yaml
-     - id: mypy
-       types_or: [ python, pyi ]
-       args: ["--ignore-missing-imports", "--scripts-are-modules"]
+      - id: mypy
+        types_or: [python, pyi]
+        args: ["--ignore-missing-imports", "--scripts-are-modules"]
```

Comment:
Where do these changes come from? Did you run a formatter over this? (Same with the actions YAML files)